Documentation/Events/Events/Frontend/AfterLinkIsGeneratedEvent.rst

```diff
@@ -0,0 +1,69 @@
+.. include:: /Includes.rst.txt
+.. index:: Events; FilterMenuItemsEvent
+.. _FilterMenuItemsEvent:
+
+====================
+FilterMenuItemsEvent
+====================
+
+.. versionadded:: 12.0
+   This PSR-14 Event supersedes the :php:`UrlProcessorInterface` logic
+   which allowed to modify mail URNs or external URLs, but not the
+   full anchor tag.
+
+   In addition, this PSR-14 event also replaces the
+   :php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['tslib/class.tslib_content.php']['typoLink_PostProc']`
+   hook which was not executed at all times, and had a cumbersome API
+   to modify values.
+
+   It is also recommended to use the PSR-14 Event instead of the global
+   getATagParams hook (:php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['tslib/class.tslib_content.php']['getATagParamsPostProc']`)
+   to add additional attributes (see example below) to links.
+
+This event allows PHP developers to modify any kind of link generated
+by TYPO3's mighty :php:`typolink()` functionality.
+
+By using this event, it is possible to add attributes to links to
+internal pages, or links to files, as the Event contains the actual information
```

Comment:
In the sentence, once "event" is with lower "e", once with upper "E". Is there a rule?